project/plugins.sbt

```diff
@@ -3,3 +3,4 @@ addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "1.1.0")
 addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.4.3")
 addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.9.30")
 addSbtPlugin("com.geirsson" % "sbt-ci-release" % "1.5.7")
+addSbtPlugin("com.thoughtworks.sbt-api-mappings" % "sbt-api-mappings" % "3.0.0")
```

Comment:
Wow I'm surprised that this is even a problem in the first place 😯

Reply:
As in, there's no equivalent of javadoc's `@link` which allows for pointing to any class from the doc?

Reply:
You can only ever point to the same class's members from scaladoc?

Reply:
You can also point to the members of standard library.
But I want to point to members of `io.circe` and `scala.reflection`.